examples/aasp/aasp-sample-template.yaml

```diff
@@ -20,6 +20,9 @@ spec:
     volumeMounts:
     - mountPath: /opt/confidential-containers/share/kata-containers/reference-info-base64
       name: endorsement-location
+    ports:
+      - containerPort: 8080
+      - containerPort: 50000
```

Comment:
The default should not expose AASP outside of the pod.

Reply:
I'm not sure what needs to be done here.

Reply:
Just remove the port declarations for the aasp container

Reply:
done